astropy/utils/iers/tests/test_iers.py

```diff
@@ -149,9 +149,16 @@ def test_simple(self):
         assert status3 == iers.FROM_IERS_A_PREDICTION
         assert ut1_utc3 != 0.
 
+
 class TestIERS_Auto():
 
     @remote_data
```

Comment:
In order to make the test *fail* without the patched code, I had to put in `remote_data`.  I.e. the exact same test code would fail when run interactively in ipython but pass in the test. There is something in the testing environment and some interaction that I didn't feel like chasing down, but `remote_data` did the trick.